### PR TITLE
PageContent loading state

### PR DESCRIPTION
**Pull request by dgalarza**
image

I think we can eventually make this even nicer and utilize React's `<Suspense>` but right now Apollo doesn't support it.

https://github.com/apollographql/apollo-client/issues/10231

### CODE REVIEW

File: packages/components/page_template/src/page_content.tsx [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
   children,
+  loading,
   ...props
 }): React.ReactElement => {
   return (
-    <Box gridArea="content" mt="40px" px="page.gutterX" {...props}>
-      {children}
+    <Box
+      gridArea="content"
+      display={loading ? "grid" : "block"}
```
Comment (acostanzo) [inline packages/components/page_template/src/page_content.tsx:18]: Should this be: 
```suggestion
      display={loading ? "flex" : "block"}
```

**Review by acostanzo — COMMENTED** [review]
I think this is cool! There also seems to be a decent path for abstraction if we need to leverage this loader shim pattern in other contexts than the whole page.

File: packages/components/page_template/src/page_content.tsx (hunk 1 above)
Comment (dgalarza) [reply packages/components/page_template/src/page_content.tsx:18]: Both work, not sure if one is better than the other.

Comment (dgalarza) [reply packages/components/page_template/src/page_content.tsx:18]: Actually, turns out if I use `display: flex` then `justify-self` doesn't work on the box below.

Per MDN:

> In flexbox layouts, this property is ignored. Read more about [alignment in Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Alignment/Box_Alignment_in_Flexbox).

**Review by tute — APPROVED** [review]

**Pull request merged**